ext/CreateParametersExt.jl

```diff
@@ -259,7 +260,12 @@ end
 
 EnergyHydrologyParameters has two constructors: float-type and toml dict based.
 Additional parameters must be added manually: ν, ν_ss_om, ν_ss_quartz, ν_ss_gravel, hydrology_cm, K_sat, S_s, and θ_r
-All parameters can be manually overriden via keyword arguments.
+All parameters can be manually overriden via keyword arguments. Note, however,
+that certain parameters must have the same type (e.g, if a field is 
+supplied for porosity, it must be supplied for all other parameters
+defined in the interior of the domain). Some parameters are defined only
+on the surface of the domain (e.g albedo), while other are defined everywhere
+(e.g. porosity). These are indicated with types `F` and `SF`.
```

Comment:
It would be good to include somewhere in the documentation a list of surface parameters and a list of subsurface parameters

Reply:
good idea! I added a description in the doc string for EnergyHydrologyParameters.